charts/agent/README.md

```diff
@@ -63,57 +64,60 @@ The command removes all the Kubernetes components associated with the chart and
+| `slim.kmoduleImage.repository`          | The kernel module image builder repository to pull from                                                                                    | `sysdig/agent-kmodule`                                      |
+| `slim.kmoduleImage.digest`              | The image digest to pull                                                                                                                   | ` `                                                         |
+| `slim.resources.requests.cpu`           | CPU requested for building the kernel module                                                                                               | `1000m`                                                     |
+| `slim.resources.requests.memory`        | Memory requested for building the kernel module                                                                                            | `348Mi`                                                     |
+| `slim.resources.limits.memory`          | Memory limit for building the kernel module                                                                                                | `512Mi`                                                     |
+| `ebpf.enabled`                          | Enable eBPF support for Sysdig instead of `sysdig-probe` kernel module                                                                     | `false`                                                     |
+| `ebpf.settings.mountEtcVolume`          | Needed to detect which kernel version are running in Google COS                                                                            | `true`                                                      |
+| `clusterName`                           | Set a cluster name to identify events using *kubernetes.cluster.name* tag. Overrides `global.clusterConfig.name`                           | ` `                                                         |
+| `sysdig.accessKey`                      | Your Sysdig Agent Access Key. Overrides `global.sysdig.accessKey`                                                                          | ` ` Either accessKey or existingAccessKeySecret is required |
+| `sysdig.existingAccessKeySecret`        | Alternatively, specify the name of a Kubernetes secret containing an 'access-key' entry. Overrides `global.sysdig.existingAccessKeySecret` | ` ` Either accessKey or existingAccessKeySecret is required |
+| `sysdig.disableCaptures`                | Disable capture functionality (see https://docs.sysdig.com/en/disable-captures.html)                                                       | `false`                                                     |
+| `sysdig.settings`                       | Additional settings, directly included in the agent's configuration file `dragent.yaml`. Overrides other settings.                         | `{}`                                                        |
```

Comment:
Along the lines of the helper change, since I took out the message, I just noted here that anything specified with `sysdig.settigns` will override other values.